### PR TITLE
fix: ensure logServerAddress gets an actual URL

### PR DESCRIPTION
**Pull request by jlipps**
The server was failing to start entirely. How was CI passing before?

### CODE REVIEW

**Review by boneskull — APPROVED** [review]

File: packages/appium/lib/main.js [hunk 1]
```diff
@@ -442,7 +442,7 @@ async function main(args) {
     });
   }
 
-  logServerAddress(`${parsedArgs.address}:${parsedArgs.port}${parsedArgs.basePath}`);
+  logServerAddress(`http://${parsedArgs.address}:${parsedArgs.port}${parsedArgs.basePath}`);
```
Comment (mykola-mokhnach) [inline packages/appium/lib/main.js:445]: not sure if it is a good idea to hardcode `http`. We could use `parsedArgs.scheme` in order to make it more flexible/error prone

Comment (jlipps) [reply packages/appium/lib/main.js:445]: yeah good idea. i was looking for some arg like this but couldn't find it. will update.

Comment (jlipps) [reply packages/appium/lib/main.js:445]: actually @mykola-mokhnach i grepped the codebase for `scheme` but don't see it as an appium arg?

Comment (mykola-mokhnach) [reply packages/appium/lib/main.js:445]: ah, looks like that [PR](https://github.com/appium/appium/pull/18562) has never been merged. Sorry for the confusion

**Pull request merged**